config/md5

```diff
@@ -948,6 +948,7 @@ rubygems-2.2.0.rc.1.tgz=f7d80b612339169569f2675155c202f1
 rubygems-2.2.0.tgz=3c16e50673adb99d1ce76d5231f764bd
 rubygems-2.4.8.tgz=dc77b51449dffe5b31776bff826bf559
 rubygems-2.6.8.tgz=40b3250f28c1d0d5cb9ff5ab2b17df6e
+rubygems-2.6.9.tgz=61dc2378bbb20a875049a86844bcd84c
```

Comment:
Instead of adding new entry you should replace existing one. See https://github.com/rvm/rvm/commit/340d0475e2d784ba36854272328aa3ce92282118